meetings/2025-03-05.md

```diff
@@ -0,0 +1,121 @@
+* eP: It would be interesting to see how it plays in local-first setup. Where VCs would be replicaterd.
+* HZ: Do you know that the open wallet Inrupt donated to OWF is white labeled?
+* MdJ: I'm using API but not building the app.
+* HZ: We want to make it pluggable on the backend. On the frontend you may want to change what is being displayed etc.
+
+### PATCH
+* MdJ: The way I see it, https://github.com/solid/specification/issues/715 sounds like it would not be a minor bugfix, so I think it would be more a task for LWS WG, not for Solid CG.
+* MdJ: Regarding https://github.com/solid/specification/issues/715 I would initially say it sounds like a minor bugfix, but after reading about how difficult this would be for implementors, given the apparent unavailability of off-the-shelf parsers that could deal with this, I don't think it's within remit for Solid CG to merge a PR like this, so again, this feels to me like something we should leave for LWS WG to work on.
+* MdJ: Regarding textual changes, yes, I think they are necessary and I think we can work on them in GitHub PRs and discuss them in Solid  CG calls. In particular it seems there is one mention of blank nodes in a place where blank nodes are not allowed, we should probably just remove that mention: https://github.com/solid/specification/pull/716
+* JW: I see a few issues conflated. The discussion Ruben had, about no off the shelf parsers. The issue is that currently N3 doesn't define how blank notes scoping works. It is undefined if two blank nodes with the same label in different graphs are co-reffering or distinct.
+* JW: For N3 patch this isn't much of an issue. We don't propose that deletions are allowed to refer to blank node. Besides the patch identifier itself there are no blank nodes in the document. We can propose a compromise that bypasses all of the issue mentioned by Ruben.
+* JW: Dorthe have commented that there is an interest in N3 WG that would resolve that issue with blank node scoping. We can procede with the insertion and be aware with the pitfalls in current parsers.
+* MdJ: Have anyone implemented it yet?
+* JW: NSS always allowed it, CSS forbids it. CSS uses N3 which can parse those blank nodes, there is just one line in CSS that checks if the blank nodes exist. I understand Ruben's argument but I see it as academic precission.
+* MdJ: If NSS already does it, if CSS doesn't do it we can do it in Pivot. ESS doesn't do N3 patch at all.
+* eP: Do you know why ESS doesn't support it?
+* MdJ: The whole patching is in the air. When the switch away from SPARQL update happended, it was known that WG will be resolving that. So this wasn't a priority to implement what was proposed in CG.
+* RG: Why is it that we put N3 patch as specification inside solid protocol spec instead having it as a normative reference. Jesse can you make that comment about planned change in the PR as a note?
```

Comment:
```suggestion
* MdJ: The whole area of patching is in the air. When the switch away from SPARQL Update happened, it was known that WG would be resolving that, so it wasn't a priority to implement what was proposed in CG.
* RG: Why is it that we put N3 Patch as a specification inside the Solid Protocol spec instead of having it as a normative reference? Jesse, can you make that comment about planned change in the PR as a note?
```